README.md.gotmpl

```diff
@@ -1,39 +1,33 @@
-# Coder Helm Chart
+# Coder Helm Chart &middot; [![build](https://github.com/cdr/enterprise-helm/actions/workflows/build.yml/badge.svg)](https://github.com/cdr/enterprise-helm/actions/workflows/build.yml) [![Twitter Follow](https://img.shields.io/twitter/follow/CoderHQ?label=%40CoderHQ&style=social)](https://twitter.com/coderhq)
```

Comment:
just a personal opinion, I think it's better to put these on their own line below the header line, because Markdown uses the header name to generate a link, and so the link to this header has some extraneous hyphens: `https://github.com/cdr/enterprise-helm/blob/34040562829bd72f9091bb955032ddd0ac4b89ee/README.md#coder-helm-chart---`

I also am not sure whether it matters, but I recently changed the monorepo README to refer to a specific event (`push`) and branch (`master`) in the markup: `[![coder](https://github.com/cdr/m/actions/workflows/coder.yaml/badge.svg?event=push&branch=master)](https://github.com/cdr/m/actions/workflows/coder.yaml?query=branch%3Amaster)`

Reply:
I'm good with that.

The default branch is used when one isn't specified, but I've updated the specific event!